server/test/views/components/SelectWithLabelTest.java

```diff
@@ -48,6 +48,13 @@ public void createSelect_rendersSelectedOption() {
     assertThat(selectWithLabel.getSelectTag().render()).doesNotContain("<optgroup>");
   }
 
+  @Test
```

Comment:
suggestion (non-blocking): I don't know that you need this test since `SelectWithLabel` is using the `focusOnError` method on `FieldWithLabel` (if I am understanding correctly). I think it's enough to have the `FieldWithLabel` test.